### PR TITLE
Update Glia Core SDK to 0.20.0

### PR DESCRIPTION
**Pull request by gugalo**
The audio permission has been fixed in the Glia Core SDK version 0.20.0. Fix consists of adding new `queueForEngagement`.

An additional change that has been made during these changes is related to Bintray closing its services.

> Update [July 4, 2021]
> Thanks for supporting Bintray! This service has now been sunset.

https://status.bintray.com/

### CODE REVIEW

File: widgetssdk/build.gradle [hunk 1]
```diff
@@ -62,4 +62,30 @@ dependencies {
     // removed because becomes slow on Android 11. They say it is fixed, but I still have the issue.
     // https://square.github.io/leakcanary/changelog/
     // debugImplementation "com.squareup.leakcanary:leakcanary-android:$leakCanaryVersion"
+
+    // Replaced with a local AAR file
+//    implementation "com.glia:android-sdk:$gliaSdkVersion"
+    // Glia sdk aar
+    implementation(name: 'glia-core-sdk-0.20.0', ext: 'aar')
```
Comment (liinahamari) [inline widgetssdk/build.gradle:69]: Mb better to use variable use have declared before? $gliaSdkVersion

**Review by liinahamari — APPROVED** [review]

File: widgetssdk/build.gradle (hunk 1 above)
Comment (gugalo) [reply widgetssdk/build.gradle]: Fixed: https://github.com/salemove/android-sdk-widgets/compare/7734b2d3066197a01a15942ea951e793a6c34266..a27372cd5fbfc29655174516012ff7ea88e9471e

File: app/build.gradle [hunk 2]
```diff
@@ -61,7 +61,7 @@ ext {
     constraintLayoutVersion = '2.0.4'
     navVersion = '2.3.1'
     preferenceVersion = '1.1.1'
-    gliaSdkVersion = '0.17.0'
+    gliaSdkVersion = '0.20.0'
```
Comment (deniss-mc) [inline app/build.gradle:64]: You can remove it here - its example app deps

File: app/build.gradle [hunk 3]
```diff
@@ -75,6 +75,7 @@ dependencies {
     implementation "androidx.navigation:navigation-ui:$navVersion"
     implementation "androidx.preference:preference:$preferenceVersion"
     implementation project(path: ':widgetssdk')
+    implementation(name: "glia-core-sdk-$gliaSdkVersion", ext: 'aar')
```
Comment (deniss-mc) [inline app/build.gradle:78]: You can remove it here - its example app deps

**Review by deniss-mc — CHANGES_REQUESTED** [review]
LGTM - but you forgot to include the aar- file 

Use
git add --force FILEPATH

**Comment by gugalo** [comment]
@deniss-mc AAR file added: https://github.com/salemove/android-sdk-widgets/compare/a27372cd5fbfc29655174516012ff7ea88e9471e..f2d3855110c85ddab2681f5c76e6fe7850bc7b4e

**Review by deniss-mc — APPROVED** [review]
LGTM

**Comment by gugalo** [comment]
I have tested it many times. But the engagement request does not reach the operator on the first attempt after a clean install of the app.

**Comment by gugalo** [comment]
To be more specific. Engagement request is not sent to the operator if permissions have been not granted in advance. Issue reproducible only in the Widget SDK example app

**Comment by gugalo** [comment]
So I have been investigating the issue and found out that for some reason even though the permissions are getting requested correctly from the Core SDK side. For some reason, once the permissions have been granted (or even declined) the Widget SDK Example app method `onRequestPermissionsResult` is not getting triggered. 

I first tried to debug it but failed. As this fix is needed ASAP I have created fast and dirty implementation to handle the permissions directly from the Widget SDK CallACtivity.

**Comment by gugalo** [comment]
!merge

**Pull request merged**